src/lib.rs

```diff
@@ -426,9 +426,16 @@ mod tests {
     // and any associated usability regressions before a new Rust version is shipped
     // but will have more stable results than Nightly.
     #[rustversion::beta]
+    #[cfg(feature = "enable-static-tests")]
     #[test]
     fn static_test() { static_test_impl() }
```

Comment:
If I wanted to run just this test, I would use the following command, right?

```sh
cargo test --features enable-static-tests static-test
```

Reply:
Right, exactly.

Reply:
Maybe I should add a compile error if someone enables the feature with a Rust toolchain other than beta:

```rust
#[rustversion:not(beta)]
#[cfg(feature = "enable-static-tests")]
compile_error!("The `enable-static-tests` feature can only be enabled with a beta Rust toolchain");
```

Reply:
Added that check.